save-backend/src/main/kotlin/com/saveourtool/save/backend/service/ProjectService.kt

```diff
@@ -82,38 +81,80 @@ class ProjectService(
 
     /**
      * @param organizationName
+     * @param status
      * @return List of the Organization projects
      */
-    fun getAllByOrganizationName(organizationName: String) = projectRepository.findByOrganizationName(organizationName)
+    fun getAllByOrganizationNameAndStatus(organizationName: String, status: ProjectStatus? = null) = status?.let {
+        projectRepository.findByOrganizationNameAndStatus(organizationName, status)
+    } ?: projectRepository.findByOrganizationName(organizationName)
 
     /**
      * @param organizationName
+     * @param status
      * @return Flux of the Organization projects
      */
-    fun getAllAsFluxByOrganizationName(organizationName: String) = getAllByOrganizationName(organizationName).let { Flux.fromIterable(it) }
+    fun getAllAsFluxByOrganizationNameAndStatus(organizationName: String, status: ProjectStatus? = null) =
+            getAllByOrganizationNameAndStatus(organizationName, status).let { Flux.fromIterable(it) }
 
     /**
      * @return project's without status
      */
     fun getNotDeletedProjects(): List<Project> {
         val projects = projectRepository.findAll { root, _, cb ->
-            cb.notEqual(root.get<String>("status"), ProjectStatus.DELETED)
+            cb.notEqual(root.get<String>("status"), ProjectStatus.CREATED)
         }
         return projects
     }
 
+    /**
+     * @param project
+     * @return [project] with status [ProjectStatus.CREATED]
+     */
+    fun recoverProject(project: Project) = changeProjectStatus(project, ProjectStatus.CREATED)
+
+    /**
+     * @param project
+     * @param status
+     * @return [project] with status [ProjectStatus.DELETED] or [ProjectStatus.BANNED]
+     */
+    fun deleteProject(project: Project, status: ProjectStatus) = changeProjectStatus(project, status)
```

Comment:
Let's separate this method into `banProject` and `deleteProject`, which will call `changeProjectStatus(project, ProjectStatus.DELETED)` and `changeProjectStatus(project, ProjectStatus.BANNED)`